testexec/defaults.go

```diff
@@ -46,3 +50,28 @@ func defaultAssertFn(t *testing.T, actual, expect string) {
 		t.Errorf("expected: %q; actual: %q", expect, actual)
 	}
 }
+
+// Hash Table of all the "special" nodes used by testmark.
+var DefaultLeafNodes = map[string]struct{}{
```

Comment:
Exporting this seems unnecessary.  I can't conceive of a reason someone should be able to modify this, nor even a good reason to want to inspect it.

Reply:
Sure